cmd/reviewdog.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/stevemcquaid/mcq/pkg/commands"
+)
+
+var ReviewDogCmd = &cobra.Command{
+	Use:   "reviewdog",
+	Short: "-> reviewdog",
+	Long:  `Runs reviewdog`,
+	Run: func(cmd *cobra.Command, args []string) {
+		_ = commands.ReviewDog(PRNumFlag)
+	},
+}
+
+var PRNumFlag int
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported var PRNumFlag should have comment or be unexported